spec/models/spree/adjustment_spec.rb

```diff
@@ -61,38 +61,43 @@ module Spree
       describe "Shipment adjustments" do
         let(:shipping_method) { create(:shipping_method_with, :flat_rate) }
         let(:shipment)        { create(:shipment_with, :shipping_method, shipping_method: shipping_method) }
-        let!(:order)          { create(:order, distributor: hub, shipments: [shipment]) }
+        let(:order)           { create(:order, distributor: hub) }
         let(:hub)             { create(:distributor_enterprise, charges_sales_tax: true) }
-        let!(:line_item)      { create(:line_item, order: order) }
+        let(:line_item)       { create(:line_item, order: order) }
         let(:adjustment)      { order.adjustments(:reload).shipping.first }
 
-        it "has a shipping charge of $50" do
-          adjustment.amount.should == 50
+        describe "the shipping charge" do
+          it "is the adjustment amount" do
+            order.shipments = [shipment]
+
+            adjustment.amount.should == 50
```

Comment:
Please, use RSpec3's `expect`.

Reply:
I am going to transpec'ize and rubocup'ize :-D this file when it's all green (when this, 2812 and 2816 are merged). Ok?

Reply:
here: #2856